.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: false
+language: ruby
+cache: bundler
+rvm:
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
+script:
+  - export RAILS_ENV=test
+  - bundle exec rake
```

Comment:
If you use the `env:` section as I've suggested above then I think you can get rid of this line as it's the default for `language: ruby`.